src/utils.js

```diff
@@ -41,3 +41,45 @@ export function quote(str) {
 export function unquote(str) {
   return str.replace(/^"/, "").replace(/"$/, "");
 }
+
+/**
+ * Sorts records in a list according a given ordering.
+ * @param  {String} ordering The ordering.
+ * @param  {Array}  list     The collection to order.
+ * @return {Array}
+ */
+export function sortObjects(order, list) {
+  const hasDash = order[0] === "-";
+  const field = hasDash ? order.slice(1) : order;
+  const direction = hasDash ? -1 : 1;
+  return list.slice().sort((a, b) => {
+    if (!a[field] || !b[field])
+      return 0;
```

Comment:
I would say that if a value is defined for this field in one of the two records, it comes first (undefined comes after)